src/search_algo.hpp

```diff
@@ -1354,686 +661,6 @@ _writeRecord(TBlastRecord & record,
 // Function computeBlastMatch()
 // --------------------------------------------------------------------------
 
-template <typename TBlastRecord,
```

Comment:
Why don't we need this huge block anymore?

Reply:
It's the old "hand-crafted" back-tracking algorithm. it is replaced by standard search now.

Later-on is the "hand-crafted" seed-and-extend approach which I have gotten rid of now since it won't ever work with SeqAn3 and is so heuristic, it's almost random 😉

Reply:
I remember that we talked about it :smile: